environment/container/kubernetes/kubeconfig.go

```diff
@@ -42,6 +43,10 @@ func (c kubernetesRuntime) provisionKubeconfig(ctx context.Context) error {
 	})
 
 	kubeconfFile := filepath.Join(hostKubeDir, "config")
+	envKubeConfFile := c.host.Env("KUBECONFIG")
+	if envKubeConfFile != "" {
+		kubeconfFile = strings.Split(envKubeConfFile, string(os.PathListSeparator))[0]
```

Comment:
Actually there is an in-built function for this `filepath.SplitList` https://pkg.go.dev/path/filepath@go1.22.2#SplitList.